src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -131,6 +131,22 @@ protected override void ProcessRecord()
                             }
                         }
 
+                        //RootModule can be null, empty string or point to a valid .psm1, or .dll.  Anything else is invalid.
```

Comment:
I think the extension comparison issue should probably be a separate issue so we can apply same fix everywhere